src/repository/photos.py

```diff
@@ -0,0 +1,67 @@
+from typing import List, Type
+from sqlalchemy import and_
+from sqlalchemy.orm import Session
+from src.database.models import Photo, User, Tag
+from src.schemas import PhotoModel, PhotoResponse, TagModel, TagsPhoto
+from datetime import date, timedelta
+
+from fastapi import UploadFile
+
+
+async def create_photo(text: str, url: str, db: Session) -> Photo:
+
+    photo = Photo(
+        user_id=1,
```

Comment:
Need to fix the logic with constant value

Reply:
this I can make when we have user role. for the moment authorization is added, but confirmation email is empty and user cannot authorise to load the photos